extensions/aproc/proc/access/storages/file.py

```diff
@@ -0,0 +1,51 @@
+import enum
+import os
+import shutil
+from pathlib import Path
+from typing import Literal
+from urllib.parse import urlparse
+
+from extensions.aproc.proc.access.storages.abstract import AbstractStorage
+
+
+class AccessType(enum.Enum):
+    READ = "read"
+    WRITE = "write"
+
+
+class FileStorage(AbstractStorage):
+    type: Literal["file"] = "file"
+    authorized_paths: list[str]
+    writable: bool
+
+    def get_storage_parameters(self):
+        return {}
+
+    def supports(self, href: str):
+        scheme = urlparse(href).scheme
+        return scheme == "" or scheme == "file"
```

Comment:
Important: `href` must start with one of the `authorized_paths` to consider that this file storage supports the path (href).